scripts/auto-rebase/handle-assets.py

```diff
@@ -19,18 +26,23 @@
 
 
 def merge_paths(pathl, pathr):
+    """Merge two paths into one"""
```

Comment:
The first time I looked at this function I thought we should replace it with os.path.join, so adding a little detail about how it is different from that function would be useful.

```suggestion
    """Merge two paths into one by removing any '/' prefix from pathr and then appending it to pathl"""
```

Reply:
Thanks for the suggestion, addressed.